packages/fiber/src/core/renderer.ts

```diff
@@ -46,87 +49,83 @@ interface Catalogue {
   }
 }
 
-// https://github.com/mrdoob/three.js/issues/21209
-// HMR/fast-refresh relies on the ability to cancel out props, but threejs
-// has no means to do this. Hence we curate a small collection of value-classes
-// with their respective constructor/set arguments
-const DEFAULT = '__default'
-const defaultMap = new Map()
-;[
-  [THREE.Box2, [new THREE.Vector2(+Infinity, +Infinity), new THREE.Vector2(-Infinity, -Infinity)]],
-  [THREE.Box3, [new THREE.Vector2(+Infinity, +Infinity), new THREE.Vector2(-Infinity, -Infinity)]],
-  [THREE.Color, ['white']],
-  [THREE.Cylindrical, [1, 0, 0]],
-  [THREE.Euler, [0, 0, 0, 'XYZ']],
-  [THREE.Vector2, [0, 0]],
-  [THREE.Vector3, [0, 0, 0]],
-  [THREE.Vector4, [0, 0, 0, 1]],
-  [THREE.Matrix3, [1, 0, 0, 0, 1, 0, 0, 0, 1]],
-  [THREE.Matrix4, [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1]],
-  [THREE.Quaternion, [0, 0, 0, 1]],
-  [THREE.Sphere, [new THREE.Vector3(), -1]],
-  [THREE.Ray, [new THREE.Vector3(), new THREE.Vector3(0, 0, -1)]],
-  [THREE.Spherical, [1, 0, 0]],
-  [THREE.Triangle, [new THREE.Vector3(), new THREE.Vector3(), new THREE.Vector3()]],
-].forEach(([type, args]) => defaultMap.set(type, args))
-
```

Comment:
did we drop this?

Reply:
Yeah we wanted to move it for now & come up with a better way to do it. There's a property called memoizedProps, I think paul had an idea of doing something with it.

Reply:
is it still this block thats removed? bc defaultMap is being accessed in the original. i think memoprops wouldnt solve removed props because it has to store on the target object (vectors, colors, etc). for now i would still leave it in until we find a better way.

Reply:
I've reverted this change